tools/link.py

```diff
@@ -1946,7 +1946,7 @@ def run_embind_gen(wasm_target, js_syms, extra_settings):
   outfile_js = in_temp('tsgen_a.out.js')
   # The Wasm outfile may be modified by emscripten.run, so use a temporary file.
```

Comment:
```suggestion
  # The Wasm outfile may be modified by emscripten.emscript, so use a temporary file.
```

There's one more reference to emscripten.run above as well, but GH won't let me add a suggestion for that line.